_studio/shared/umc/core/vm_plus/include/umc_mutex.h

```diff
@@ -40,28 +40,6 @@ class Mutex : public MfxMutex
     Mutex& operator=(const Mutex&);
```

Comment:
what about ```MfxAutoMutex```, ```Mutex```, ```MfxMutex``` classes? Do we use them in some other code?

Reply:
yes, we still do